spec/timezone.html

```diff
@@ -546,6 +546,27 @@ <h1>FormatISOTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-objectimplementstemporaltimezoneprotocol" type="abstract operation">
+      <h1>
+        ObjectImplementsTemporalTimeZoneProtocol (
+          _object_: an Object,
+        ): either a normal completion containing a Boolean, or an abrupt completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It determines whether the given _object_ is a Temporal.TimeZone instance or implements the required methods from the Temporal.TimeZone protocol.
+          It will always return a normal completion unless _object_ is a Proxy or has one in its prototype chain.
```

Comment:
This sounds like "always throws for proxies". Not sure how to make it better.

Reply:
Maybe something like "For ordinary objects, and some exotic objects, this operation is infallible. However, if _object_ is a Proxy or has one in its prototype chain, it may call into user code."